src/helpers/glpTEST/getGlpSwapData.js

```diff
@@ -0,0 +1,319 @@
+import { ethers, providers } from "ethers";
+import { swap0xRequest } from "@/helpers/0x";
+import { actions } from "@/helpers/cauldron/cook/actions";
+
+import gmxVaultAbi from "@/helpers/glpTEST/abi/gmxVault";
+import gmxLensAbi from "@/helpers/glpTEST/abi/gmxLens";
+import { priceAbi } from "@/utils/farmPools/abi/priceAbi";
+const gmxVaultAddress = "0x489ee077994B6658eAfA855C308275EAd8097C4A";
+const gmxLensAddress = "0x9f8fB63ef774A496cd9aD3Cc29c1e81Ae947FC2e";
+import store from "@/store";
+
+const blacklis = [
+  "0x17fc002b466eec40dae837fc4be5c67993ddbd6f",
+  "0xfea7a6a0b346362bf88a9e4a88416b77a57d6c2a",
+  "0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9",
```

Comment:
```suggestion
  "0xFa7F8980b0f1E64A2062791cc3b0871572f1F7f0",
```
USDT should not be blacklisted, but UNI should be